great_expectations/data_context/data_context_variables.py

```diff
@@ -71,6 +71,12 @@ def __post_init__(self) -> None:
         if self.substitutions is None:
             self.substitutions = {}
 
+    def __str__(self) -> str:
+        return str(self.config)
+
+    def __repr__(self) -> str:
+        return repr(self.config)
```

Comment:
If a class has no `__str__` it will use `__repr__`, do we need both?

Reply:
nope! I always get confused by these - let me remove `__str__`